app/models/ems_event.rb

```diff
@@ -338,6 +338,29 @@ def ems_refresh_target
     ext_management_system
   end
 
+  def src_container_replicator
+    entity = container_replicator_id && ContainerReplicator.find_by(:id => container_replicator_id)
+    return entity if entity
+    _log.warn("Unable to find target ContainerReplicator for event: #{inspect}")
+    nil
+  end
+
+  def src_container_group
+    entity = container_group_id && ContainerGroup.find_by(:id => container_group_id)
+    return entity if entity
+    _log.warn("Unable to find target ContainerGroup for event: #{inspect}")
+    nil
+  end
+
+  def src_container_node
+    # TODO: resolve functional duplication with process_container_entities_in_event!
+    #       (https://github.com/ManageIQ/manageiq/issues/9600)
```

Comment:
The query below is working around #9600; once `process_container_entities_in_event!` is fixed, this should use `container_node_id`.

Doing a similar query in `process_container_entities_in_event!` would "fix" #9600.
But it's not as "reliable" as a real uid, if we can make k8s/kubeclient send it.  Might need to fall back on a name query when uid is missing.  Didn't want to block this PR on figuring out the right fix...